### ISSUE TITLE
2 `<main>` tags

### ISSUE THREAD

**Issue by yamcodes** [yamcodes/arkenv#615]
We have 2 `<main>` tags on the homepage, one includes the Header, one doesn't.

This violates the WHATWG HTML living standard, so we should ideally add a test (in our a11y suite) that makes sure our docs pages (we can take an arbitrary one as a sample) and our home page each have a single `<main>` tag within them.
